index.html

```diff
@@ -2,17 +2,36 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Name of Tab Goes Here</title>
+    <title>Tale of the Cart that Turns into a Dragon</title>
   </head>
 
   <body>
     <h1>Cart Turns Into Dragon</h1>
     <b>Let's build something</b>
-    <p>
+    <br>
+    <br>
+    <br>
+    <h2>A story...</h2>
+    <p style="color:green;font-family:arial">Garrett the gecko was a true believer and a dreamer.</p>
```

Comment:
For line 15, I set the font color to green and the font family to Arial. I used http://www.cssfontstack.com/ to look at different font families I could use.